be/src/storage/rowset/column_reader.h

```diff
@@ -213,7 +213,7 @@ class ColumnReader {
 
     uint8_t _flags = 0;
     // counter to record the reader's mem usage, sub readers excluded.
-    std::atomic<size_t> _meta_mem_usage = 0;
+    std::atomic<size_t> _meta_mem_usage;
```

Comment:
The default constructor is trivial: no initialization takes place other than zero initialization of static and thread-local objects. https://en.cppreference.com/w/cpp/atomic/atomic/atomic

Reply:
good catch. intend to do it in constructor, but uses fetch_add and misses explicit initialization.